documentation/modules/ROOT/pages/lca.adoc

```diff
@@ -0,0 +1,48 @@
+= Lifecycle Agent Operator (LCA)
+include::_attributes.adoc[]
+:profile: 5g-ran-lab
+
+The Lifecycle Agent Operator (LCA) provides local (on cluster) lifecycle management services for Single Node OpenShift (SNO). These management services include the following operations:
+
+* Seed Image generation for Image Based Installations and Image Based Upgrades.
+* Image Based Upgrades Orchestration.
+* Backup and Restore orchestration during Image Based Upgrades.
+
+[#seed-image-generation]
+== Seed Image Generation
+
+The seed image generation requires of an existing SNO cluster, this cluster is called _seed cluster_. The SNO cluster needs to follow specific guidelines to be used as seed cluster, you can check the full list {lca-seed-cluster-guidelines}[here].
+
+Some of the guidelines include:
+
+* Share hardware configuration as the rest of the nodes that will be installed with the generated Seed Image.
+* Shared `/var/lib/containers` partition.
+
+When the LifeCycle Agent Operator runs the seed image generation, the orchestrator will run its system config validation checks to ensure the required seed SNO configuration is present. In case there is some blocking configuration, the relevant information will be stored in the `SeedGenerator` custom resource.
+
+[#image-based-installation]
+== Image Based Installation
+
+The role of LCA during IBI deployments is just the generation of the seed image that will be used during the final install. As we will see in future sections, IBI deployments require booting the hardware with a _pre-installation iso_ that leverages the seed image. 
+
+
+[#image-based-upgrades]
+== Image Based Upgrades
+
+The role of LCA during IBU operations includes the generation of the seed image that will be used during the upgraded, as well as the orchestration of the upgrade process itself, including backup and restore of cluster configurations and running workloads.
+
+During the upgrade, the LCA will orchestrate the process during the following stages:
+
+* Idle
+* Prep
+* Upgrade
+* Rollback (Optional)
+
```

Comment:
I would include the picture from docs where it is clearly shown the different stages: https://docs.openshift.com/container-platform/4.18/edge_computing/image_based_upgrade/cnf-understanding-image-based-upgrade.html#cnf-image-based-upgrade_understanding-image-based-upgrade

Reply:
Added